MNE/disp3D/materials/shadermaterial.cpp

```diff
@@ -103,23 +103,23 @@ ShaderMaterial::ShaderMaterial(QNode *parent)
 ShaderMaterial::~ShaderMaterial()
 {
     //Not sure if Qt3d module implemented internal garbage handling, so I do it manually here
-    delete m_pAmbientParameter;
-    delete m_pDiffuseParameter;
-    delete m_pSpecularParameter;
-    delete m_pShininessParameter;
-    delete m_pAlphaParameter;
-    delete m_pFilterKey;
+//    delete m_pAmbientParameter;
+//    delete m_pDiffuseParameter;
+//    delete m_pSpecularParameter;
+//    delete m_pShininessParameter;
+//    delete m_pAlphaParameter;
+//    delete m_pFilterKey;
```

Comment:
why are the member not deleted anymore, are those paramter QObjects - consider guarding them with QPointer -> they get automatically destroyed